oap-server/server-storage-plugin/storage-influxdb-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/influxdb/query/MetadataQuery.java

```diff
@@ -125,7 +125,7 @@ public Service searchService(String serviceCode) throws IOException {
         final WhereQueryImpl<SelectQueryImpl> whereQuery = select(ID_COLUMN, NAME, ServiceTraffic.GROUP)
             .from(client.getDatabase(), ServiceTraffic.INDEX_NAME)
             .where(eq(TagName.NODE_TYPE, String.valueOf(NodeType.Normal.value())));
-        whereQuery.and(eq(ServiceTraffic.NAME, serviceCode));
+        whereQuery.and(eq("\"" + ServiceTraffic.NAME + "\"", serviceCode));
```

Comment:
We have a constant with quotes, `InfluxConstants.NAME`.

Reply:
thanks,i submit again.